SQRLDotNetClientUI/Views/MainMenuView.xaml

```diff
@@ -81,7 +81,7 @@
 
       <!-- Action buttons -->
       <Button Content="{loc:Localization BtnNewIdentity}" Name="btnNewIdentity" Command="{Binding OnNewIdentityClick}" Height="30" Width="130" Margin="10" Grid.Row="1" Grid.Column="0" HorizontalAlignment="Center" />
-      <Button Content="{loc:Localization BtnRekeyIdentity}" Name="btnReKeyIdentity" Command="{Binding RekeyIdentity}" Height="30" Width="130" Margin="10" Grid.Row="1" Grid.Column="1" IsEnabled="{Binding CurrentIdentityLoaded}" HorizontalAlignment="Left"/>
+      <Button Content="{loc:Localization BtnRekeyIdentity}" Name="btnReKeyIdentity" Command="{Binding RekeyIdentity}" Height="30" Width="130" Margin="10" Grid.Row="1" Grid.Column="1"  HorizontalAlignment="Left"/>
```

Comment:
I guess you've removed this for testing, shall we put it in again?

Reply:
Yup, fixed